locals.tf

```diff
@@ -6,56 +6,44 @@ locals {
     integration    = false
     preprod        = false
     production     = false
-    management     = true
```

Comment:
you should be able to remove this `is_mgmt_env ` block now as it looks like it might be unused

Reply:
removed